### PR TITLE
Upgrading styled components

### PR DESCRIPTION
**Pull request by SamGodwin2**
### What is the context of this PR?
Updated styled components and rewrote a test or two. 

### How to review 
Give the entire app a once over, make sure all the styles are the same as before and nothings messed up

### CODE REVIEW

**Comment by SamGodwin2** [comment]
[Upgrade react to 16.11](https://trello.com/c/C4NygDpL/1282-upgrade-react-to-1611)

**Comment by heofs** [comment]
Screenshot 2019-12-05 at 11 40 39
😭

File: eq-author/src/App/QuestionnairesPage/QuestionnairesView/QuestionnairesTable/Row/index.js [hunk 1]
```diff
@@ -8,7 +8,6 @@ import CustomPropTypes from "custom-prop-types";
 
 import IconButtonDelete from "components/buttons/IconButtonDelete";
 import DuplicateButton from "components/buttons/DuplicateButton";
-import FadeTransition from "components/transitions/FadeTransition";
```
Comment (heofs) [inline eq-author/src/App/QuestionnairesPage/QuestionnairesView/QuestionnairesTable/Row/index.js]: Why no more fade transition on deleting questionnaire?

Comment (SamGodwin2) [reply eq-author/src/App/QuestionnairesPage/QuestionnairesView/QuestionnairesTable/Row/index.js]: Well spotted 😉 I couldn't get it to work in a reasonable way, there was a slightly breaking change and getting it to work was starting to take longer than was worth it, especially as you never really see it due to the modal being in the way.

Comment (heofs) [reply eq-author/src/App/QuestionnairesPage/QuestionnairesView/QuestionnairesTable/Row/index.js]: Okay thats fine. I think we should have a session where we decide what animations etc we should have, as I dont think we have any consistent use of them. It also needs design work to decide what the animation should do.

**Pull request merged**